crates/sui-core/src/authority.rs

```diff
@@ -4387,6 +4409,9 @@ impl AuthorityState {
         if let Some(tx) = self.create_bridge_tx(epoch_store) {
             txns.push(tx);
         }
+        if let Some(tx) = self.init_bridge_committee_tx(epoch_store) {
+            txns.push(tx);
+        }
```

Comment:
in this way, can we still make bridge committee tx in the same epoch of bridge creation? It's an ergonomics for local tests

Reply:
I think we can, by passing a flag to `init_bridge_committee_tx` indicating the intention to create a bridge.

Reply:
this cannot run at the same time with the bridge creation because it takes the bridge object as an input, also we need to wait for the validators to call the registration before we can start the committee initiation, the committee will be created at the end of the next epoch,  i.e. for a new network, the bridge object is created at genesis, committee init will try to run starting from end of epoch 0; for a existing network, framework updated at end of epoch x, the bridge object will be created at end of epoch x, committee init will run at the end of epoch x+1.

We could call the init committee function manually before the end of epoch tx to make this nicer for local test

Reply:
resolved offline